package.json

```
@@ -38,5 +38,11 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Cloud-CV/EvalAI.git"
+  },
+  "devDependencies": {
+    "jasmine-core": "^3.4.0",
```

```suggestion
    "jasmine-core": "^3.4.0",
    "angular-mocks": "^1.7.0",
```

Lets keep angular-mocks inside dev-dependencies rather than dependency.